local_store.go

```diff
@@ -304,7 +317,7 @@ func (f *fileSystemStore) Commit(consistentSnapshot bool, versions map[string]in
 	return f.Clean()
 }
 
-func (f *fileSystemStore) GetSigningKeys(role string) ([]sign.Signer, error) {
+func (f *fileSystemStore) GetSigningKeys(role string) ([]keys.Signer, error) {
```

Comment:
Can we rename this to GetSigners/Signers?   --- because these are Signers we are returning, basically?

Reply:
Would be a big API change -- WDYT? Yeah, they're the Signers, which are signing keys

Reply:
Are we breaking anyone? -- Do we know if any external system using this directly?

Reply:
btw, you are already changing the method's signature.

Reply:
Good point :P I think the only person that breaks is me in some sigstore code :dagger: 
So i'm happy to do so -- we also don't have a 1.0 release so I'm not too worried about conflicts

Reply:
done